build.sh

```diff
@@ -50,6 +51,25 @@ check_managed_prereqs()
     fi
 }
 
+install_dotnet_cli()
+{
+    echo "Installing the dotnet/cli..."
+    hash wget 2>/dev/null || { echo >&2 "Please ensure curl is present before running this script"; exit 1; }
```

Comment:
@schellap How does the CLI repo perform the download of their packages? do they expect curl installed?